theme.toml

```diff
@@ -1,15 +1,15 @@
 # theme.toml template for a Hugo theme
 # See https://github.com/spf13/hugoThemes#themetoml for an example
 
-name = "Hemingway"
+name = "Hemingway2"
 license = "MIT"
-licenselink = "https://github.com/tanksuzuki/hemingway/blob/master/LICENSE.md"
+licenselink = "https://github.com/beli3ver/hemingway2/blob/master/LICENSE.md"
 description = "Really minimal blog theme for hugo"
-homepage = "https://github.com/tanksuzuki/hemingway"
+homepage = "https://github.com/beli3ver/hemingway2"
 tags = ["blog", "font awesome", "highlight.js"]
-features = ["html5", "css3"]
+features = ["html5", "css3", "tags"]
 min_version = 0.16
 
 [author]
-  name = "tanksuzuki"
-  homepage = "http://tanksuzuki.com"
+  name = "Malte Kiefer"
```

Comment:
I'm not very familiar with the evolution of this theme, but how does removing references to the original author make sense for a PR (with a completely unrelated title as well)?